web/packages/core/package.json

```diff
@@ -9,15 +9,35 @@
         "pkg/"
     ],
     "scripts": {
-        "build": "npm run build:cargo && npm run build:wasm-bindgen && npm run build:wasm-opt && npm run build:ts",
-        "build:cargo": "cross-env-shell \"cargo build --release --target wasm32-unknown-unknown --features \\\"$CARGO_FEATURES\\\"\"",
+        "build": "npm run build:ruffle_web && npm run build:ruffle_web_fancy && npm run build:ts",
+
+        "//1": "# Setting RUFFLE_REAL_FANCY_WASM=true will result in the 'fancy' module being built properly, instead of just being a 'fake', a copy of the basic one.",
+        "build:ruffle_web": "npm run build:cargo && npm run build:wasm-bindgen && npm run build:wasm-opt",
+        "build:ruffle_web_fancy": "node -e \"(process.env.RUFFLE_REAL_FANCY_WASM == 'true') ? process.exit(0) : process.exit(1)\" && npm run build:ruffle_web_fancy_real || npm run build:ruffle_web_fancy_fake",
+
+        "build:ruffle_web_fancy_real": "echo \"Making real fancy module\" && npm run build:cargo_fancy && npm run build:wasm-bindgen_fancy && npm run build:wasm-opt_fancy",
+        "build:ruffle_web_fancy_fake": "echo \"Making fake fancy module\" && shx cp ./pkg/ruffle_web_bg.wasm ./pkg/ruffle_web_fancy_bg.wasm && shx cp ./pkg/ruffle_web_bg.wasm.d.ts ./pkg/ruffle_web_fancy_bg.wasm.d.ts && shx cp ./pkg/ruffle_web.js ./pkg/ruffle_web_fancy.js && shx cp ./pkg/ruffle_web.d.ts ./pkg/ruffle_web_fancy.d.ts",
```

Comment:
Not happy about how complex it looks like, but I don't see a nice cross-platform way of writing it either :(

Reply:
Honestly, same... :|

Reply:
Well, except for what @relrelb just commented, using `cross-env-shell`!